rest/handler.go

```diff
@@ -215,23 +216,32 @@ func (h *handler) invoke(method handlerMethod, accessPermissions []string, respo
 	}
 
 	// If an Admin Request we need to check the user credentials
-	if h.shouldCheckAdminAuth() {
+	if (h.privs == adminPrivs && *h.server.config.API.AdminInterfaceAuthentication) || (h.privs == metricsPrivs && *h.server.config.API.MetricsInterfaceAuthentication) {
 		username, password := h.getBasicAuth()
+		if username == "" {
+			if dbContext == nil || dbContext.Options.SendWWWAuthenticateHeader == nil || *dbContext.Options.SendWWWAuthenticateHeader {
+				h.response.Header().Set("WWW-Authenticate", wwwAuthenticateHeader)
+			}
+			return base.HTTPErrorf(http.StatusUnauthorized, "Invalid login")
```

Comment:
I think this is login required, rather than invalid?

```suggestion
			return base.HTTPErrorf(http.StatusUnauthorized, "Login Required")
```